src/core/ee/cop1.hpp

```diff
@@ -41,6 +41,7 @@ class Cop1
         bool get_condition();
 
         uint32_t get_gpr(int index);
+        float get_gpr_f(int index);
```

Comment:
Recommend marking function as `const`.

```suggestion
        float get_gpr_f(int index) const;
```